frontend/components/site/sitePublishedBranchesTable.js

```diff
@@ -62,7 +64,7 @@ class SitePublishedBranchesTable extends React.Component {
   }
 
   renderLoadingState() {
-    return <p>No published branch data available</p>
+    return <LoadingIndicator/>
```

Comment:
Is it ever possible to have no published branch data here (we loaded it and it's empty)?